File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0503Q00FilteringTheInformationInfo.java
package nta.med.data.model.ihis.ocsa;

import java.sql.Timestamp;

public class OCS0503Q00FilteringTheInformationInfo {
	private String reserYn;
    private String jinryoTime;        
    private String gwaName;
    private String doctorName;
    private String naewonYn;
    private String bunho;
    private Timestamp naewonDate;
    private String gwa;
    private String doctor;
    private String naewonType;
    private Double jubsuNo;
	public OCS0503Q00FilteringTheInformationInfo(String reserYn,
			String jinryoTime, String gwaName, String doctorName,
			String naewonYn, String bunho, Timestamp naewonDate, String gwa,
			String doctor, String naewonType, Double jubsuNo) {
		super();
		this.reserYn = reserYn;
		this.jinryoTime = jinryoTime;
		this.gwaName = gwaName;
		this.doctorName = doctorName;
		this.naewonYn = naewonYn;
		this.bunho = bunho;
		this.naewonDate = naewonDate;
		this.gwa = gwa;
		this.doctor = doctor;
		this.naewonType = naewonType;
		this.jubsuNo = jubsuNo;
	}
	public String getReserYn() {
		return reserYn;
	}
	public void setReserYn(String reserYn) {
		this.reserYn = reserYn;
	}
	public String getJinryoTime() {
		return jinryoTime;
	}
	public void setJinryoTime(String jinryoTime) {
		this.jinryoTime = jinryoTime;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getNaewonYn() {
		return naewonYn;
	}
	public void setNaewonYn(String naewonYn) {
		this.naewonYn = naewonYn;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public Timestamp getNaewonDate() {
		return naewonDate;
	}
	public void setNaewonDate(Timestamp naewonDate) {
		this.naewonDate = naewonDate;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getNaewonType() {
		return naewonType;
	}
	public void setNaewonType(String naewonType) {
		this.naewonType = naewonType;
	}
	public Double getJubsuNo() {
		return jubsuNo;
	}
	public void setJubsuNo(Double jubsuNo) {
		this.jubsuNo = jubsuNo;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0503Q00FilteringTheInformationInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0503Q00FilteringTheInformationInfo
	{
		private String _reserYn;
		private String _jinryoTime;
		private String _gwaName;
		private String _doctorName;
		private String _naewonYn;
		private String _bunho;
		private String _naewonDate;
		private String _gwa;
		private String _doctor;
		private String _naewonType;
		private String _jubsuNo;

		public String ReserYn
		{
			get { return this._reserYn; }
			set { this._reserYn = value; }
		}

		public String JinryoTime
		{
			get { return this._jinryoTime; }
			set { this._jinryoTime = value; }
		}

		public String GwaName
		{
			get { return this._gwaName; }
			set { this._gwaName = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String NaewonYn
		{
			get { return this._naewonYn; }
			set { this._naewonYn = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String NaewonDate
		{
			get { return this._naewonDate; }
			set { this._naewonDate = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String NaewonType
		{
			get { return this._naewonType; }
			set { this._naewonType = value; }
		}

		public String JubsuNo
		{
			get { return this._jubsuNo; }
			set { this._jubsuNo = value; }
		}

		public OCS0503Q00FilteringTheInformationInfo() { }

		public OCS0503Q00FilteringTheInformationInfo(String reserYn, String jinryoTime, String gwaName, String doctorName, String naewonYn, String bunho, String naewonDate, String gwa, String doctor, String naewonType, String jubsuNo)
		{
			this._reserYn = reserYn;
			this._jinryoTime = jinryoTime;
			this._gwaName = gwaName;
			this._doctorName = doctorName;
			this._naewonYn = naewonYn;
			this._bunho = bunho;
			this._naewonDate = naewonDate;
			this._gwa = gwa;
			this._doctor = doctor;
			this._naewonType = naewonType;
			this._jubsuNo = jubsuNo;
		}

	}
}